### PR TITLE
MUMUP-2000 : Change stuff

### PR DESCRIPTION
**Pull request by timlevett**
+ "{{TITLE}} Apps" changed names to "Search and Browse {{TITLE}}"
+ Sidebar updates from search to Browse with new FA icon
+ Mobile only shows "Search {{TITLE}}"
+ Changed list and widget to compact and expanded
+ Updated URLs as well for list/widget, note people that were widget will be redirected to compact
+ Not changing apps URL due to complexity
+ I'm not doing screenshots, but here is the doc: https://docs.google.com/presentation/d/1Bhi-XQ6HvQnPMyGko0L9tqdH2fNkULsygvOAhy0Vbf0/edit#slide=id.ga49f3c057_7_42

### CODE REVIEW

File: angularjs-portal-frame/src/main/webapp/js/app-config.js [hunk 1]
```diff
@@ -3,10 +3,6 @@ define(['angular'], function(angular) {
     var config = angular.module('app-config', []);
     config
         .constant('APP_FLAGS', {
-            'enableToggle': true,
-            'defaultView' : 'list',
-            'list' : true,
-            'widgets' : true,
```
Comment (timlevett) [inline angularjs-portal-frame/src/main/webapp/js/app-config.js]: these are not needed in frame so removed

File: angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
     </div>
   </div>
   <div class="show-all-div">
-    <p>Showing {{ (portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length }} of {{ portlets.length }} apps. </p>
+    <p>Showing {{ (portlets | filter:searchTermFilter | showApplicable:showAll | showCategory:categoryToShow).length }} of {{ portlets.length }} MyUW. </p>
```
Comment (vertein) [inline angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace.html]: Showing x of x `results in` MyUW.

**Comment by vertein** [comment]
:+1:

**Comment by jhanstra** [comment]
:+1:

File: angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html [hunk 3]
```diff
@@ -1,22 +1,22 @@
 <span class="layout-chooser" ng-if="toggle" ng-class="{ 'hidden-xs' : !$storage.mobileWidgetToggle}">
-  <li ng-class="{active : modeIs('list')}">
-    <a  href="javascript:;" ng-click="switchMode('list')"
-      tooltip-html-unsafe="Icons"
+  <li ng-class="{active : modeIs('compact')}">
```
Comment (apetro) [inline angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html]: Would making this `{active : !modeIs('expanded')}` catch any edge cases where mode is somehow neither `compact` nor `expanded`?

**Pull request merged**